packages/react/src/utilities/RovingFocus/RovingFocusItem.tsx

```diff
@@ -52,12 +52,24 @@ export const RovingFocusItem = forwardRef<HTMLElement, RovingFocusItemProps>(
         const items = getOrderedItems();
         let nextItem: RovingFocusElement | undefined;
 
-        if (e.key === 'ArrowRight') {
+        if (e.key === 'ArrowRight' || e.key === 'ArrowDown') {
           nextItem = getNextFocusableValue(items, focusValue);
+          e.preventDefault();
         }
 
-        if (e.key === 'ArrowLeft') {
+        if (e.key === 'ArrowLeft' || e.key === 'ArrowUp') {
```

Comment:
This has an unintentional side-effect of focus moving left and right on up/down arrow keys on `Tabs` and `ToggleGroup` which are horisontal. Pretty sure this should not be the case.

Edit: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

Reply:
I thought this should always be the case, but we can make it a prop.